modules/test/playwright/fixtures/featureFlagsTest.ts

```diff
@@ -0,0 +1,171 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {Page, mergeTests, test} from '@playwright/test';
+
+import {loginTest} from './loginTest';
+
+export type FeatureFlagsOptions = {
+	[key: string]: boolean;
+};
+
+export type FeatureFlags = FeatureFlag[];
+
+export interface FeatureFlag {
+	readonly enabled: boolean;
+	readonly key: string;
+}
+
+function featureFlagsTest(options: FeatureFlagsOptions) {
+	const fixtureImpl = test.extend<{
```

Comment:
Sneaky? I am not sure if fixtures were really intended to be defined this way, as plain old functions, but it doesn't really matter. It allows for a very nice API to set FFs :+1:

Reply:
I assume it's perfectly valid, since the way the framework is constructed seems to be aligned with this usage. But I agree it's a bit "advanced" :sweat_smile: 

I considered using [fixture options](https://playwright.dev/docs/test-fixtures#fixtures-options) but they are too coarse grained for our needs, I think.